CRM/Member/Form/Membership.php

```diff
@@ -1184,13 +1182,8 @@ public function submit() {
         $this->_id,
         'Membership'
       );
-      $membershipTypes[$memType] = CRM_Core_DAO::getFieldValue('CRM_Member_DAO_MembershipType',
-        $memType
```

Comment:
Just noting for the record the default param for $returnColumn in this function is name